snap/snapcraft.yaml

```diff
@@ -73,9 +79,10 @@ parts:
       ./usr/share/srsran/sib.conf.example: ./usr/share/srsran/sib.conf
       ./usr/share/srsran/user_db.csv.example: ./usr/share/srsran/user_db.csv
       ./usr/share/srsran/rb.conf.example: ./usr/share/srsran/rb.conf
-    override-build: |
-      set -x
-      pushd $SNAPCRAFT_PART_SRC
-      sed -ie 's/native/x86_64/g' CMakeLists.txt
-      popd
-      snapcraftctl build
+    override-prime: |
+      snapcraftctl prime
+      sed -i '/device_args = tx_port=tcp/s/^#//g' ./usr/share/srsran/ue.conf
+      sed -i '/device_name = zmq/s/^#//g' ./usr/share/srsran/ue.conf
+      sed -i '/device_args = fail_on_disconnect/s/^#//g' ./usr/share/srsran/enb.conf
+      sed -i '/device_name = zmq/s/^#//g' ./usr/share/srsran/enb.conf
+
```

Comment:
I know this is somewhat unrelated to this PR but we should probably change the default log file as well

Reply:
Do you have a suggestion where to log? :) we can include it here, no problem.

Reply:
It will log to  `/var/snap/srsran/current/` by default.
Done.